namespace py Const

typedef i64 TaskId

enum Tasks {
    AllWishes = 0
    CharacterWishes = 1,
    NoviceWishes = 2,
    StandardWishes = 3,
    WeaponWishes = 4,
}